tpm2/tpm2_test.go

```diff
@@ -285,6 +294,33 @@ func TestHash(t *testing.T) {
 	}
 }
 
+func supportsECC(rw io.ReadWriter) (bool, error) {
+	var err error
+	moreData := true
```

Comment:
only `moreData` should be outside the loop
`for i := uint32(0); moreData; i++ {`

Reply:
Much nicer, thanks.